homeassistant/components/zha/core/channels/lighting.py

```diff
@@ -44,6 +44,7 @@ class ColorChannel(ZigbeeChannel):
         "color_temp_physical_max": True,
         "color_capabilities": True,
         "color_loop_active": False,
+        "start_up_color_temperature" : True,
```

Comment:
Remove the space before the colon. If you install the precommit hook it will catch these issues. You can find instructions in the HA developer docs.

Reply:
Sorry, that commit was from a non-dev environment.

Reply:
No need to apologize!